ember-toucan-core/src/components/form/controls/checkbox.ts

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+export interface ToucanFormCheckboxControlComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    isDisabled?: boolean;
+    onChange?: (
+      isChecked: boolean,
+      e: Event | InputEvent,
+      isIndeterminate: boolean
```

Comment:
Would this be ever true? Like when you click an indeterminate checkbox, it will get either checked or unchecked, but the `indeterminate` property will always turn to false by the browser, won't it?

Also, this status is a bit redundant, as we already have the event, so a user could just ask for `e.target.indeterminate`. Same goes for `isChecked` of course, but that is what people will want to use in 99,9% of times, so it's reasonable to pass that as the primary value of `onChange`. But for `isIndeterminate` I am thinking whether we can drop it here from the list of agruments? 🤔

Reply:
Yeah, maybe so! I was looking at some existing implementation and that's why I added this here, but you're right.  Maybe we just remove it altogether and if folks need to look at `indeterminate` they can pull it off of `e.target` as you mention.  That may make things a bit easier overall.